### ISSUE TITLE
can not run apt-get udpate - probably chris-lea repo related 

### ISSUE THREAD

**Issue by muzykalukasz** [TalkingQuickly/rails-server-template#59]
can not run apt-get udpate on the server after using this repo to bootstap it. 

Issue seems to be related to chris-lea repository.

Error when running apt-get update: 

W: The repository 'http://ppa.launchpad.net/chris-lea/node.js/ubuntu xenial Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: Failed to fetch http://ppa.launchpad.net/chris-lea/node.js/ubuntu/dists/xenial/main/binary-amd64/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.

**Comment by danielristic** [comment]
Hello,

It really is a provisioning issue. Like explained [here](https://github.com/TalkingQuickly/rails-server-template/issues/64) some of the dependencies are a bit outdated and you might end up with a functional but partially buggy setup.

Here, the issue is related to nodejs and the dependency issue is introduced in `rails_gem_dependencies-tlq`, which is solved in this [pull request](https://github.com/TalkingQuickly/rails_gem_dependencies-tlq/pull/4) that has not yet been merged.

To remedy it, you need to use the forked repository instead by changing the line in Berksfile to:
```
cookbook 'rails_gem_dependencies-tlq', github: 'flyingboy007/rails_gem_dependencies-tlq'
```

This basically installs nodejs by requiring the `ark` cookbook, but ark is dependent on `seven_zip` and its latest version needs a newer version of Chef that break other stuff (while you should be using version 12), so you also need to lock seven zip to an older version: 

```
cookbook 'seven_zip', '2.0.2'
```

Once you've added these to the Berksfile, simply `berks update rails_gem_dependencies-tlq` and then, following the regular provisionning instructions you should end up with a node can apt-get update without errors.

**Comment by TalkingQuickly** [comment]
Thanks, @danielristic is spot on, the whole cookbook has been updated to support the latest Chef version and cookbooks as well as Ubuntu 20.04 (the new LTS since 16.04 reaches EOL April 30th 2021). So this is all now fixed in the 5.0.1 release. I'm currently working on a CI setup for this repo so that the end to end process including installing chef tools and then doing a full server build on a fresh VM can be run so that I'll get notified in future as soon as things break.
